### PR TITLE
Small updates

### PR DESCRIPTION
**Pull request by lawrencehlee**
- Some stylistic changes to TT show
- About page updates
- README update

### CODE REVIEW

**Comment by lawrencehlee** [comment]
[README update](https://trello.com/c/tYOw8LwC/148-readme-update)

**Comment by lawrencehlee** [comment]
[Remove briefs from top ten show pages](https://trello.com/c/b1e3xfx9/141-remove-briefs-from-top-ten-show-pages)

**Comment by lawrencehlee** [comment]
[Add contribution link to about page](https://trello.com/c/ToixbqZx/164-add-contribution-link-to-about-page)

File: app/views/static_pages/about.html.erb [hunk 1]
```diff
@@ -26,6 +28,18 @@
       on <a class="author-link" href="https://github.com/lawrencehlee/themq-website">Github</a>.
       </p>
 
+      <h2 class="paragraph-header">Donating</h2>
+      <p class="actual-paragraph">
+      We've come to a point in which, after 20 years of operation, not all our printing costs are covered
```
Comment (BarakTzori) [inline app/views/static_pages/about.html.erb]: So this isn't technically true - they used to sell ad space

**Comment by lawrencehlee** [comment]
[No dividing sidebar line in top ten show](https://trello.com/c/DC7QIAEA/163-no-dividing-sidebar-line-in-top-ten-show)

**Pull request merged**